doc: Updating Bluetooth Qualification page (#28398)
nrfconnect/sdk-nrf: doc/nrf/protocols/bt/bt_qualification/index.rst

   :local:
   :depth: 2

Every product using *Bluetooth®* technology must be qualified through a process defined by the Bluetooth Special Interest Group (SIG).
When your product is qualified, you will be authorized to do the following:

* Prove that your products satisfy the requirements of the Bluetooth license agreements and specifications.
* Use Bluetooth trademark and logos.
* Sell products that use Bluetooth LE technology.

> As a result, your products will be publicly listed and can be promoted on the official Bluetooth SIG page.

Reviewer: ```suggestion
As a result, your products will be [publicly listed](https://www.bluetooth.com/qualified-products/) and can be promoted on the official Bluetooth SIG page.
```

If relevant, you can use an existing design that has already been qualified (called a *pre-qualified design*) and has a Bluetooth Design Number (DN) or, for older qualifications, a Qualification Design ID (QDID) number.
This allows you to speed up your product's qualification process by omitting any additional testing.
You will notice that all valid DNs begin with the letter Q while older QDID numbers do not have a Q prefix to the number.

If your product uses a qualified Bluetooth stack provided as part of the |NCS|, you can inherit this qualification from Nordic Semiconductor, provided that you do not introduce any changes to the stack.

To find the applicable DNs, go to the Compatibility Matrix relevant for the SoC or SiP model you use (for example, the `nRF52840 Compatibility Matrix`_), and open the Bluetooth DNs and QDIDs section.

You can independently qualify any qualified design offered by Nordic Semiconductor by performing all of the required testing yourself.
However, this process might be complex and time-consuming.
Thus, it is recommended that you inherit Nordic Semiconductor's design qualifications in your product's listing.

> If you want to have additional Bluetooth profiles in your product design, you must run the required tests through the `Profile Tuning Suite (PTS)`_ tool before qualifying the product.

Reviewer: ```suggestion
If you want to have additional Bluetooth profiles (or additional ICS layers) in your product design, that are not offered through Nordic Semiconductor's design qualifications, you must implement them by yourself and run the required tests through the `Profile Tuning Suite (PTS)`_ tool before qualifying the product. 
```

Reviewer: @mileslouissmith  what do you think of this suggestion?

Requirements
************

Before you start the Bluetooth SIG's Qualification Process, ensure the following:

1. You have access to your company's member account.
   To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.

#. Get acquainted with the Qualification Program Reference Document (QPRD) and Declaration Process Document (DPD) to understand the basics of the qualification process.
   Log into your company's member account to review both the QPRD and DPD documents.

Qualification process
*********************

> Although you can make use of Nordic's pre-qualified designs, you need to perform the Bluetooth Product qualification for your product yourself.

Reviewer: ```suggestion
Although you can make use of Nordic's pre-qualified designs, you need to perform online paperwork necessary for initiating the Bluetooth Product qualification of your product, yourself.
```

Hence, you must complete the qualification process for all of your products under your company name and account.

> You can either perform a full qualification process for a new product or edit an existing entry of a product that has already been qualified.

Reviewer: ```suggestion
You can either perform a full qualification process for a new product or edit an existing entry of a product that has already been qualified by your company.
```

See the following sections for more information about both possibilities.

.. note::

   Each new qualification listing requires a fee from the Bluetooth SIG for the DN.
   At the top of the main page for the Qualification Workspace tool, you will find a link marked **Pay Product Qualification Fee**.
   You can pay the fee for a receipt number using invoice or credit card.
   Your company must have at least one available receipt number to complete a new product listing.

Creating a new product listing
==============================

Log into the Qualification Workspace (found at the `Bluetooth SIG's Qualification Process`_ page), the official qualification tool , that will guide you through one of the following processes depending on the state of your product:

> * Qualification Option 2a: Create a new design that combines multiple existing designs.

Reviewer 2: ```suggestion
* Qualification Option 2a - Create a new design that combines multiple existing designs.
```

  Complete it if your product uses only pre-qualified DNs and implements APIs as instructed in the corresponding |NCS| documentation.
  Refer to the QPRD section 3.2.2.1 for Option 2a.
  This is the most common and straightforward path for Nordic customers.

> * Qualification Option 2b: Create any other new design.

Reviewer 2: ```suggestion
* Qualification Option 2b - Create any other new design.
```

  Complete it in the following cases:

>   * If your product uses a new Bluetooth design that is not yet qualified by Nordic Semiconductor.

Reviewer: Through out this doc, it is not fully clear what exactly the 'Bluetooth design' is? Perhaps define this in the top? Because this concept 'Bluetooth design' is fundamental to the qualification process. Customers can qualify their unimplemented ICS layer by Nordic, by implementing themselves and running tests. So, the design in that case would be that specific ICS layer. In general Bluetooth design is the stack, firmware, chips, etc. provided by Nordic. You can also link the definition of the '[design](https://www.bluetooth.com/wp-content/uploads/Files/Specification/HTML/QPRD/out/en/index-en.html#UUID-fca9d069-3750-ff69-a58b-7668cb588eb6)' from QPRD page wherever this word appears first.

Author: Defined on line 22. Is it not enough?

  * If your product alters a previously qualified Bluetooth design by changing the code or configuring it in a way that can break the qualification requirements for the Nordic Semiconductor's qualified design.

>     Refer to the QPRD section 3.2.2.2 for Option 2b.

Reviewer: ```suggestion
    Refer to the QPRD section [3.2.2.2](https://www.bluetooth.com/wp-content/uploads/Files/Specification/HTML/QPRD/out/en/index-en.html#UUID-5b5e9584-8af4-b02c-1198-96be8369ac0b) for Option 2b.
```

It will be great to add similar links wherever possible. It makes it much more easy for any new/old customers to follow-along this guide by simultaneously referring QPRD.

For certain DNs Nordic creates design "subsets" that contain a reduced set of features from the main design.
You can combine and use these in the same manner as the DNs from the main design.
As an example, the Nordic nRF54L15 SoC might have a "main" DN that includes the Channel Sounding feature and a subset that removes that feature for products that do not require it.
Choose the DN that most closely matches your product features and requirements.

.. note::
   It is not a problem to have "unused" features in your design.
   For example, if your referenced Nordic controller design supports the 2M PHY feature, but your product does not require it, that is not a concern.

>    You do not have to use every feature claimed in the product's qualification listing, but conversely you *must* claim every feature that you use.

Reviewer: ```suggestion
   You do not have to use every feature claimed in the product's qualification listing, but conversely you *must* claim every feature that you use.
   You can add additional products in the same design listing later (without paying any extra fee), that uses previously un-used feature from inherited design.
```

This is important to mention :)

Author: Added.

Updating an existing product listing
====================================

.. note::
   An existing product listing can be edited only by the person that completed the listing.
   If you want to change the listing owner, contact SIG support.

Update an existing qualification listing through the Qualification Workspace tool (found at `Bluetooth SIG's Qualification Process`_ page) in the following cases:

* You changed an existing product's design with no changes in the product's features, hence you want to update the listing to indicate new updates or add new marketing information regarding this product.

> * You can add a new product as a new entry to an existing list of product entries as there was no design change in the new product.

Reviewer: ```suggestion
* You can add a new product as a new entry to an existing list of product entries. The new product may or may not use previously unused features from inherited design.
```

@mileslouissmith  please check my proposed wording.

Author: Added.

If relevant, you can use an existing design that has already been qualified (called a *pre-qualified design*) and has a Bluetooth Design Number (DN) or, for older qualifications, a Qualification Design ID (QDID) number.
This allows you to speed up your product's qualification process by omitting any additional testing.
You will notice that all valid DNs begin with the letter Q while older QDID numbers do not have a Q prefix to the number.

If your product uses a qualified Bluetooth stack provided as part of the |NCS|, you can inherit this qualification from Nordic Semiconductor, provided that you do not introduce any changes to the stack.

To find the applicable DNs, go to the Compatibility Matrix relevant for the SoC or SiP model you use (for example, the `nRF52840 Compatibility Matrix`_), and open the Bluetooth DNs and QDIDs section.

You can independently qualify any qualified design offered by Nordic Semiconductor by performing all of the required testing yourself.
However, this process might be complex and time-consuming.
Thus, it is recommended that you inherit Nordic Semiconductor's design qualifications in your product's listing.

> If you want to have additional Bluetooth profiles (or additional ICS layers) in your product design that are not offered through Nordic Semiconductor's design qualifications, you must run the required tests through the `Profile Tuning Suite (PTS)`_ tool before qualifying the product.

Reviewer: ```suggestion
If you want to have additional Bluetooth profiles (or additional ICS layers) in your product design that are not offered through Nordic Semiconductor's design qualifications, you must implemented them and run the required tests through the `Profile Tuning Suite (PTS)`_ tool before qualifying the product.
```
